guillotina/traversal.py

```diff
@@ -399,9 +400,16 @@ def set_root(self, root: Optional[IApplication]):
         try:
             result = await self.real_resolve(request)
         except (response.Response, aiohttp.web_exceptions.HTTPException) as exc:
+            _clean_request(request, exc)
```

Comment:
This is already doing here: https://github.com/plone/guillotina/pull/785/files#diff-4decc550a3956a0cf87e7f196cac7d40L365